CONTRIBUTING.md

```diff
@@ -122,3 +122,22 @@ from the main (upstream) repository:
 [github]: https://github.com/gdg-x
 [pluspage]: https://google.com/+GDGXProject
 [js-style-guide]: http://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml
+
+## <a name="members"></a> GDG[x] contributors
+Everybody is invited to dive into the code, explore, learn, open issues, and send pull requests.
+
+### Teams
+Each project has a *team of contributors* that can merge pull requests. These team members should have a good understanding of the project to answer questions and review pull requests. These contributors are members of the corresponding github project team.
+
+One or two members are the *maintainers* of the project and publish releases. Maintainers are responsible for the road map and the state of the project in general.
```

Comment:
One or two members are the **maintainers** of the project. Maintainers are responsible for the roadmap, milestones, and the state of the project in general.  They tag and publish releases.